src/components/SwapForm/SwapModal/ConfirmSwapModalContent.tsx

```diff
@@ -53,6 +75,13 @@ const ConfirmSwapModalContent: React.FC<Props> = ({
 
   const shouldDisableConfirmButton = isBuildingRoute || !!errorWhileBuildRoute
 
+  const priceImpactFromBuild =
+    15.1 ||
```

Comment:
Redundant

Reply:
just for testing, I updated right after I push it. LOL